newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['garrett-related-products'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'a32f7cb9743197698370ee339fc94d0e430ccff3',
```

Comment:
Be careful with your keys on github!  You can setup environmental variables so that this isn't displayed.

Reply:
Ah yes. Thanks. Ill set that up.